terraform/azure_ad/terraform.tfvars

```diff
@@ -3,3 +3,10 @@ tag_project_name = "azure_ad"
 tag_production_state = "production"
 
 tag_owner_email = "jwatkins@mozilla.com"
```

Comment:
We should go through and replace jwatkins@mozilla.com through out the repo. Not needed for this PR though.